gulpfile.js

```diff
@@ -236,652 +236,320 @@ async function unpackPacks() {
 var cookErrorCount = 0;
 var cookAborted = false;
 var packErrors = {};
-async function cookPacksNoFormattingCheck() {
```

Comment:
I think you accidentally undid my changes in cook.

Reply:
I believe I fixed it